Podcast/FeedViewController.swift

```diff
@@ -28,14 +28,15 @@ class FeedViewController: ViewController {
     var feedMaxTime: Int = Int(Date().timeIntervalSince1970)
     var continueInfiniteScroll = true
     var feedSet: Set = Set<FeedElement>()
+    var emptyStateViewType: EmptyStateType = .feed
```

Comment:
shouldn't `FeedViewController`'s  empty state always be `.feed`?

Reply:
Nope. For `SharingContentViewController` in profile I do a quick hack by subclassing `FeedViewController`.